src/core/Tools.cpp

```diff
@@ -408,6 +411,21 @@ namespace Tools
         return subbed;
     }
 
+    QString cleanFilename(QString filename)
+    {
+        // Remove forward slash from title on all platforms
+        filename.replace("/", "_");
+#ifdef Q_OS_WIN
+        // Remove invalid characters on Windows
+        filename.remove(QRegularExpression("[:*?\"<>|]"));
+#endif
+#ifdef Q_OS_MACOS
+        // MacOS does not support colons
+        filename.replace(":", "");
+#endif
+        return filename.trimmed();
```

Comment:
This should probably be the same on all platforms to keep things consistent.

Reply:
Hmmmmm, I'm not entirely against it but Windows excludes so many more characters than any other OS. I didn't even add the "reserved names" COM, LPT, etc.

Reply:
I think those characters are generally to be avoided if you want any kind of cross-platform compatibility and Windows is the lowest common denominator. I don't think it's a big deal if a few more characters are replaced, but it's super annoying if you put them on a flash drive and can't even rename them any more on a Windows machine.

That said, it might be necessary to add a check whether the replacement exists and if it does, you need to add additional _ in place.